### ISSUE TITLE
Question from guide: Blockquotes, escaping special characters, and blockquote prefix

### ISSUE THREAD

**Issue by jongreig** [iandol/scrivomatic#30]
I had a question about one part of your guide, though I don't know if this is pertinent specifically to Scrivomatic. I imported the compile/styles preset from your guide, and while it works well, for block quote-styled sections of my paper (e.g.), the prefixed `> ` character in each block quote-styled paragraph gets escaped as `\> `. I currently have the 'escape characters' option set here:

<img width="889" alt="Screen Shot 2019-05-17 at 4 40 19 PM" src="https://user-images.githubusercontent.com/521658/57938083-a4c96d80-78c7-11e9-9899-1b00f55402f4.png">

And currently the prefix for the block quote style:

<img width="762" alt="Screen Shot 2019-05-17 at 5 01 16 PM" src="https://user-images.githubusercontent.com/521658/57938279-2a4d1d80-78c8-11e9-8800-1d5ef033933e.png">

It seems like Scrivener escapes whatever is in the 'prefix' field, regardless if it's not supposed to be (I would think...). Any idea to turn off escaping for this, or will I just need to disable character escaping or any Markdown transformation for this style?

**Comment by iandol** [comment]
Yes, you need to disable `Escape special characters` as it is incompatible with using Styles for generation of Markdown. In my own workflow I never activate `Convert rich text to Multimarkdown`, so it is not an issue for me. I use rich text in the editor only for comfortable visualisation linked to styles. (so the Strong style shows text bolded), but the RTF bold is ignored on compile, and the Strong style is what is transformed. 

But if you enable `Convert rich text to Multimarkdown` and disable `Escape special characters` then you may be able to mix RTF conversion and Pandoc, but be aware that things may go wrong if you use important markdown characters like [ or } etc. out of context...
